### DISCUSSION TITLE
project planning

### DISCUSSION THREAD

**Discussion by be-great** [be-great/Automsg#12] [Show and tell]
@stephenWanjala 
see the project planning: what you add as a user authentication using jwt we will consider it as another layer of security, not the main user functions and table which should contain the APIs token for both the WhatsApp and Discord.

**Comment by stephenWanjala** [comment]
Sure it's Elastic I'll update the fields to match any additional requirements

**Reply by stephenWanjala** [reply]
The  handling of login and logout
I could help set up in the front end
But sadly I'll have that part in typescript ,
Ain't big fan of JavaScript 😅

**Reply by be-great** [reply]
No worries at all! I appreciate your willingness to help with the front-end setup.
